eng/pipelines/templates/steps/npm-install.yml

```diff
@@ -11,8 +11,8 @@ steps:
   - script: npm --version --loglevel info
     displayName: npm --version
 
-  - script: npm ci
-    displayName: npm ci
+  - script: npm install --no-package-lock
```

Comment:
We don’t include this in the final pr, right?

Reply:
Correct, this is just a hack to avoid needing to update the lockfile in typespec-next.